Mathlib/RingTheory/SimpleRing/Basic.lean

```diff
@@ -79,6 +80,41 @@ lemma isField_center (A : Type*) [Ring A] [IsSimpleRing A] : IsField (Subring.ce
       _ = y * (a * (x * y)) := by rw [mul_assoc a x y]
       _ = y * a := by rw [hy, mul_one]
 
+lemma injective_ringHom_or_subsingleton_codomain
+    {R S : Type*} [NonAssocRing R] [IsSimpleRing R] [NonAssocRing S]
+    (f : R →+* S) : Function.Injective f ∨ Subsingleton S :=
+  simple.eq_bot_or_eq_top (TwoSidedIdeal.ker f) |>.recOn
+    (fun h => Or.inl <| TwoSidedIdeal.ker_eq_bot _ |>.1 h)
+    (fun h => Or.inr <| subsingleton_iff_zero_eq_one.1 <| by
+      have mem : 1 ∈ TwoSidedIdeal.ker f := h.symm ▸ ⟨⟩
```

Comment:
What's the `⟨⟩` for? Should it say `mem_top` instead?

Reply:
yes, probably. thanks